index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+	
+	<div class="nav-container">
+		<div class="wrapper">
+			<nav>
+				<div class="logo">Welcome</div>
+				<div class="menu-icon">
+					<a href="#" class="icon-btn">
+						<img src="./images/Icon - Menu.svg" alt="">
+					</a>
+				</div>
+			</nav>
+		</div>
+	</div>
+
+	<div class="background-img">
```

Comment:
- [ ] According to the project requirements, you have to use `section` to wrap the headline content.